LoyaltyProgram_Intern/src/main/java/com/pgs/soft/controller/UserController.java

```diff
@@ -78,11 +85,27 @@ public String registerView(@ModelAttribute("userDTO") UserDTO userDTO){
 	@RequestMapping(value = "/register", method=RequestMethod.POST)
 	public String register(@Valid @ModelAttribute("userDTO") UserDTO userDTO, BindingResult result){
 		if(!result.hasErrors()){
+			
+			String uuid = String.valueOf(UUID.randomUUID());
+			userDTO.setUuid(uuid);
 			userService.saveOrUpdate(userDTO);
+			String sendResult = emailService.sendConfirmationEmail("registration_conf", userDTO.getEmail(), userDTO.getEmail(), userDTO.getPassword(), uuid);
```

Comment:
please move template name to properties file or enum (try to avoid Strings in the code)